### PR TITLE
Finished up CALDP as-is docs pending comments.

### PR DESCRIPTION
**Pull request by jaytmiller**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -117,107 +116,298 @@ The abstract command for running CALDP natively is:
...
 After configuring Docker,  caldp-docker-run-pipeline runs *caldp-process* inside the docker container
-with the parameters given on the command line.
+with the parameters given on the command line.  While file: paths are defined relative to your native
+file system,  within the Docker container they will nominally be interpreted relative to */home/developer*.
+Since the CALDP_HOME directory is mounted read/write inside Docker,  files needed to process a dataset
+will be reflected back out of the Docker container to CALDP_HOME,  defaulting to your current working
+directory.
+
+**NOTE:**  Running the final cloud-like configuration above does not produce results idenitical to AWS Batch processing
+because it is only processing a single dataset and skips batch tracking and organizartion actions normally performed by
```
Comment (bhayden53) [inline README.rst:278]: typo ```organizartion```

File: README.rst [hunk 2]
```diff
@@ -117,107 +116,298 @@ The abstract command for running CALDP natively is:
...
+                <ipppssoots...>    # CALCLOUD's downloader leaves messages here, normally containing abspaths of files to archive.
+            dataset-archived/
+                <ipppssoots...>    # The archive can acknowledge archive completion here,  file contents should be preserved.
+
+Data Subdirectory
+=================
+The *data* subdirectory parallels but has a different structure than the *messages*
+subdirectory.   For every ipppssoot message,  there is a data directory and subdirectories
+which contain output files from processsing that ipppssoot.   In the current implementation,
+the ipppssoot message file is empty,  it is normally populated by the
```
Comment (bhayden53) [inline README.rst:380]: ```populated by the``` what?!?!

**Comment by bhayden53** [comment]
Typo from a previous commit :
```can be file:<relative_path> os s3://output-bucket/subdirs...```
```os -> or```

**Comment by bhayden53** [comment]
Ready to merge after fixing three typos

File: README.rst (hunk 2 above)
Comment (jaytmiller) [reply README.rst:380]: Damn.  Normally populated by calcloud.sync when it finishes downloading... with the abspaths of the files to archive.   That probably needs some work anyway but Lisa liked those.

**Pull request merged**